packages/sdk/src/networks/types.ts

```diff
@@ -0,0 +1 @@
+export type Network = "mainnet" | "testnet" | "regtest";
```

Comment:
I created this directory as I thought that the networks shouldn't be classified under `config` since a user exploring this repo may not know what config contains and this made it self-explanatory.

On a separate note, I am not sure if we will need the config folder for later for rpcs (need to discuss) but that should be an issue for later.

Reply:
I agree `config` is misleading -- something to improve in this migration but creating a separate directory just for networks looks overkill. Lets keep it as is for now and once we migrate wallets module, it can moved to `wallet/types.ts`.